services/resource-usage-tracker/tests/unit/with_dbs/test_collect_container_resource_usage_task__on_update_set.py

```diff
@@ -0,0 +1,147 @@
+import logging
+import random
+from pathlib import Path
+from unittest import mock
+
+import arrow
+import pytest
+import sqlalchemy as sa
+from faker import Faker
+from fastapi import FastAPI
+from pytest_mock import MockerFixture
+from simcore_postgres_database.models.resource_tracker import resource_tracker_container
+from simcore_service_resource_usage_tracker.resource_tracker_core import (
+    collect_container_resource_usage_task,
+)
+
+pytest_simcore_core_services_selection = [
+    "postgres",
```

Comment:
ok, so since you are using this mechanism, why is pytest-docker in your requirements? Let's remove it.